packages/core/patternlab-config.json

```diff
@@ -75,6 +75,10 @@
   "patternExportPatternPartials": [],
   "patternExportPreserveDirectoryStructure": true,
   "patternExportRaw": false,
+  "patternTranslations": {
```

Comment:
I'm not quite sure if this is a good approach but giving the user the ability to change the automated generated texts seems to be a good idea.

Reply:
We might be able to come up with a better name for what we're calling these but in general I'm totally onboard with adding support for internationalization / localization.

Reply:
I just saw that there will be an issue in the pl-nav maybe. There are some cases where checks are done against the name of the current pattern `patternSubtypeItem.patternName === 'View All'`.

Reply:
I fixed this issue in the last commit